### PR TITLE
Topic 1.1

### PR DESCRIPTION
**Pull request by NicolasRonsmans**

### CODE REVIEW

File: 01-HTML-CSS-Basics/Topic1.1/index.I.II.III.html [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    <meta name="viewport" content="width=device-width,initial-scale=1" />
+    <link rel="shortcut icon" type="image/x-icon" href="Star.ico"/>
+    <script src="http://ajax.googleapis.com/ajax/libs/jquery/1/jquery.min.js"></script>
+    <!--[íf lt IE 9]>
+	<script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><[endif] -->
+	<script type="text/javascript">
+
+ function substitute() {
+
+ 	var myValue = document.getElementById('myTextBox').value
```
Comment (NicolasRonsmans) [inline 01-HTML-CSS-Basics/Topic1.1/index.I.II.III.html]: Be sure to finish instructions with a ; at the end, to avoid bad surprises!

File: 01-HTML-CSS-Basics/Topic1.1/index.I.II.III.html [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+ 	  if (myValue.length == 0) {
```
Comment (NicolasRonsmans) [inline 01-HTML-CSS-Basics/Topic1.1/index.I.II.III.html]: Could be simplified with if (!myValue.length)

File: 01-HTML-CSS-Basics/Topic1.1/index.I.II.III.html [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+}
+th, td {
+    padding: 5px;
+}
+</style>
+</head>
+<body>
+	<header>
+    <h1 id="title">Hello World</h1>
+    <p>Escriba: My to do list y presione click</p>
```
Comment (NicolasRonsmans) [inline 01-HTML-CSS-Basics/Topic1.1/index.I.II.III.html]: You might just want headings elements being in the header. Normal text and form input would rather be outside.

File: 01-HTML-CSS-Basics/Topic1.1/index.I.II.III.html [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+	<input type="text" id="myTextBox"/> 
+	<input type="submit" Value="Click" onclick="substitute()" />
+    </header>
+    <div>
+    <h4>To do list:</h4>
```
Comment (NicolasRonsmans) [inline 01-HTML-CSS-Basics/Topic1.1/index.I.II.III.html]: Just curious, why h4 and not h2?

File: 01-HTML-CSS-Basics/Topic1.1/index.I.II.III.html [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+
+	<ul>
+	   <li>Pagar impuestos</li> 
+	   <li>Ir al gimnasio</li>
+	   <li>Cenar con amigas</li>
+	   <li>Comprar manzanas</li>
+	 </ul>
+  </div>
+  <h4>Tabla de gastos</h4>
+<table style="width:30%">
```
Comment (NicolasRonsmans) [inline 01-HTML-CSS-Basics/Topic1.1/index.I.II.III.html]: You could gather all the css rules in the same place, in the style block at line #27.

File: 01-HTML-CSS-Basics/Topic1.1/index.I.II.III.html [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+  <tr>
+    <th>Item</th>
```
Comment (NicolasRonsmans) [inline 01-HTML-CSS-Basics/Topic1.1/index.I.II.III.html]: th tags are good candidates to be placed inside a thead tag. And then, the rest can be put inside a tbody tag.

File: 01-HTML-CSS-Basics/Topic1.1/indexVI.html [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+  <meta name="description" content="Formulario con validación en js" />
+    <meta name="viewport" content="width=device-width,initial-scale=1" />
+    <link rel="shortcut icon" type="image/x-icon" href="Star.ico"/>
+    <script src="http://ajax.googleapis.com/ajax/libs/jquery/1/jquery.min.js"></script>
+    <!--[íf lt IE 9]>
+  <script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><[endif] -->
+  <script>
+function esFecha(formu)
+{
+  var elDia=formu.txt_dia.value
```
Comment (NicolasRonsmans) [inline 01-HTML-CSS-Basics/Topic1.1/indexVI.html]: Again, it works but is safer to use ;

File: 01-HTML-CSS-Basics/Topic1.1/indexVI.html [hunk 8]
```diff
@@ -0,0 +1,84 @@
...
+<input type="text" name="email" id="email"/>
+<br>
+<label>Nombre de Usuario:</label><br>
+<input type="text" name="userid" id="userid"/>
+<br>
+<label>Contraseña:</label><br>
+<input type="password" name="psw" id="pass"/>
+<br>
+<label>Deporte Favorito: </label>
+<select name="Deporte favorito">
```
Comment (NicolasRonsmans) [inline 01-HTML-CSS-Basics/Topic1.1/indexVI.html]: It would be better to use name (as well as class an ids) in lowercase, avoid spaces, dots and other special chars > "deporte-favorito"

File: 01-HTML-CSS-Basics/Topic1.1/indexVI.html [hunk 9]
```diff
@@ -0,0 +1,84 @@
...
+}
+
+</script>
+<script src="js/validation.js"></script>
+
+</head>
+<body>
+ 
+<form action="execute.php" method="post" onSubmit="return validacion();">
+<label>Nombre:</label><br>
```
Comment (NicolasRonsmans) [inline 01-HTML-CSS-Basics/Topic1.1/indexVI.html]: A "nice to have" feature, to enhance the user experience, is binding the label to its input. If your input has an id, just add a for="id-of-your-input" attribute to the label.

File: 01-HTML-CSS-Basics/Topic1.1/indexVI.html [hunk 10]
```diff
@@ -0,0 +1,84 @@
...
+   <option value="3">Natación</option>
+   <option value="4">Basquet</option> 
+   <option value="5">Golf</option> 
+   <option value="5">Otro</option> 
+</select>
+<br>
+<br>
+<label>Escribe una breve biografía sobre ti:</label><br><textarea name="bio" rows="6" cols="30">...Escribe algo sobre tí...</textarea>
+<br>
+<input type="reset" value="BORRAR"><input type="submit" id="submit" value="CREAR"/>
```
Comment (NicolasRonsmans) [inline 01-HTML-CSS-Basics/Topic1.1/indexVI.html]: Sometimes you close (<input />) your self-closing tags, sometimes not (<input>), choose which one you prefer and stick with it :)

File: 01-HTML-CSS-Basics/Topic1.1/indexVI.html [hunk 11]
```diff
@@ -0,0 +1,84 @@
...
+<input type="text" name="txt_anio" size="4"/>
+<input type="button" value="Validar fecha" onClick="alert(esFecha(this.form))"/>
+<br>
+<label>Soy:</label><br>
+<input type="radio" name="sex" value="male" checked>Hombre
+<br>
+<input type="radio" name="sex" value="female">Mujer
+<br>
+<label>E-mail</label>
+<input type="text" name="email" id="email"/>
```
Comment (NicolasRonsmans) [inline 01-HTML-CSS-Basics/Topic1.1/indexVI.html]: Just curious here, did you try with the new email input type?

File: 01-HTML-CSS-Basics/Topic1.1/index1.V.html [hunk 12]
```diff
@@ -0,0 +1,46 @@
...
+      <br>
+			<video id="video" controls preload>
+				<source src="vdo/video-prueba.mp4" type="video/mp4"/>
+   				<source src="vdo/video-prueba.webm" type="video/webm"/>
+   				<source src="vdo/video-prueba.ogv" type="video/ogg"/>
+			</video>
+		</article>
+
+
+<!--<audio controls>
```
Comment (NicolasRonsmans) [inline 01-HTML-CSS-Basics/Topic1.1/index1.V.html]: No audio?

File: 01-HTML-CSS-Basics/Topic1.1/indexVI.html (hunk 11 above)
Comment (angelestroisi) [inline 01-HTML-CSS-Basics/Topic1.1/indexVI.html]: Because I didn´t know about type email, so I write type text to be sure

File: 01-HTML-CSS-Basics/Topic1.1/indexVI.html (hunk 7 above)
Comment (angelestroisi) [inline 01-HTML-CSS-Basics/Topic1.1/indexVI.html]: I am clueless!

File: 01-HTML-CSS-Basics/Topic1.1/index.I.II.III.html (hunk 4 above)
Comment (angelestroisi) [inline 01-HTML-CSS-Basics/Topic1.1/index.I.II.III.html]: ja because h4 is smaller than h2

File: 01-HTML-CSS-Basics/Topic1.1/index.I.II.III.html (hunk 3 above)
Comment (angelestroisi) [inline 01-HTML-CSS-Basics/Topic1.1/index.I.II.III.html]: oh! yes, I forget close the header, and the <p> should be outside

File: 01-HTML-CSS-Basics/Topic1.1/index.I.II.III.html (hunk 1 above)
Comment (angelestroisi) [inline 01-HTML-CSS-Basics/Topic1.1/index.I.II.III.html]: clueless again!

File: 01-HTML-CSS-Basics/Topic1.1/index1.V.html (hunk 12 above)
Comment (angelestroisi) [inline 01-HTML-CSS-Basics/Topic1.1/index1.V.html]: the audio not work. I download a song and convert to mp3 because not work it
